deepchem/models/tf_new_models/tensor_graph.py

```diff
@@ -0,0 +1,238 @@
+import networkx as nx
+import tensorflow as tf
+import time
+
+from deepchem.models.models import Model
+from nn.copy import Layer
+
+
+class TensorGraph(Model):
```

Comment:
I'd vote for making `Sequential`, `SequentialGraph`, `SequentialSupportGraph`, `MultitaskGraphClassifier` and `MultitaskGraphRegressor` inherit from `TensorGraph`. We now have a golden opportunity for consolidating our `Tensorflow` models :-)